draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -626,18 +634,13 @@ A CA MUST NOT generate signatures over inputs that are parseable as LabeledValid
 
 [[TODO: BikeshedCertificate is a placeholder name until someone comes up with a better one. #15 ]]
 
+[[TODO: A subscriber has no way to know when a certificate expires. See #83, which avoids defining an expiration time certificate property.]]
+
 For each assertion in the tree, the CA constructs a BikeshedCertificate structure containing the assertion and a proof. A proof is a message that allows the relying party to accept the associated assertion, provided it trusts the CA and recognizes the tree head. The structures are defined below:
 
 ~~~
-enum { merkle_tree_sha256(0), (2^16-1) } ProofType;
-
 struct {
-    ProofType proof_type;
-    opaque trust_anchor_data<0..2^8-1>;
-} TrustAnchor;
-
-struct {
-    TrustAnchor trust_anchor;
+    TrustAnchorIdentifier trust_anchor;
```

Comment:
You don't define the wire format `TrustAnchorIdentifier`.

Reply:
Sorry, you do. If I understand correctly this is an ASN.1 oid and so needs byte-for-byte processing. Because of that it might be better to put after `proof_data` which is length prefixed?

Reply:
Oh yeah, that's confusing. It's intended to be `TrustAnchorIdentifier` from here:
https://davidben.github.io/tls-trust-expressions/draft-beck-tls-trust-anchor-ids.html#name-overview

Probably we should just copy-paste the definition and write it as:
```
/* See Section 4.1 of draft-beck-tls-trust-anchor-ids */
opaque TrustAnchorIdentifier<1..2^8-1>;
```

Reply:
Done.